.github/ISSUE_TEMPLATE/pipelines-scrum-template.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Pipelines Scrum Template
+about: 'Template for providing supplemental information on scrum cards. '
+title: ''
+labels: pipelines-scrum
+assignees: ''
+
+---
+
+## OUTCOME
+**What is the "done" condition of this card?**
+A clear and concise description of what the problem is.
+
+**What is the motivation for this card?**
+
+**Is there a an issue or a bug linked to this card?**
```

Comment:
I think the motivation section could capture this link, so we can drop this section.